crone/osc-methods.txt

```diff
@@ -51,9 +55,28 @@
  /set/param/cut/filter_bp [if]
  /set/param/cut/filter_br [if]
  /set/param/cut/filter_dry [if]
+ /set/param/cut/voice_sync [iif]
  /set/param/cut/pre_fade_window [if]
  /set/param/cut/rec_fade_delay [if]
  /set/param/cut/pre_fade_shape [if]
  /set/param/cut/rec_fade_shape [if]
  /set/param/cut/level_slew_time [if]
  /set/param/cut/rate_slew_time [if]
+ /set/param/cut/buffer [ii]
```

Comment:
would it be sloppy to send the second arg as a float?

otherwise i have to rewrite `o_set_cut_param` (oracle) for multiple arg types. i'm already going to add a three-arg version (for `/set/param/cut/voice_sync [iif]`)

Reply:
pretty sloppy. i already produced different `Command` constructors and `::post()` for different param lists, to avoid slop
https://github.com/catfact/norns/blob/softcut-improvements/crone/src/Commands.h#L88

Reply:
i can add the oracle glue if you wish

Reply:
agreed. i'll create different `cut_param` functions per arg types